sdk/service/service.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/mesg-foundation/engine/hash"
 	"github.com/mesg-foundation/engine/hash/dirhash"
 	"github.com/mesg-foundation/engine/service"
+	validator "github.com/mesg-foundation/engine/service/validator"
```

Comment:
```suggestion
	"github.com/mesg-foundation/engine/service/validator"
```

Reply:
For some reasons, if I remove that I have the following error which doesn't make a lof of sense
```
# github.com/mesg-foundation/engine/sdk/service
/Users/antho/prog/MESG/engine/sdk/service/service.go:17:2: service redeclared as imported package name
	/Users/antho/prog/MESG/engine/sdk/service/previous declaration at ./service.go:16:2
/Users/antho/prog/MESG/engine/sdk/service/service.go:38:31: undefined: "github.com/mesg-foundation/engine/service/validator".Service
/Users/antho/prog/MESG/engine/sdk/service/service.go:40:24: undefined: "github.com/mesg-foundation/engine/service/validator".Dependency
/Users/antho/prog/MESG/engine/sdk/service/service.go:86:12: undefined: validator
/Users/antho/prog/MESG/engine/sdk/service/service.go:99:41: undefined: "github.com/mesg-foundation/engine/service/validator".Service
/Users/antho/prog/MESG/engine/sdk/service/service.go:104:30: undefined: "github.com/mesg-foundation/engine/service/validator".Service
Error: Tests failed.
```